example_sp/djangosaml2_sp/saml2_sp/templates/amministrazione.html

```diff
@@ -0,0 +1,40 @@
+{% extends "base.html" %}
+
+{%block tab_active%} active{%endblock%}
+{% block centered_container %}
+<section id="head-section">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-6 offset-lg-1 order-lg-2">
+          <img src="https://picsum.photos/800/600" title="titolo immagine" alt="descrizione immagine" class="img-fluid">
```

Comment:
please change the following value with a static image on the local storage, using {% static "..." %}

`"https://picsum.photos/800/600"`